lib/msal-node/test/client/ManagedIdentitySources/AppService.spec.ts

```diff
@@ -176,6 +178,61 @@ describe("Acquires a token successfully via an App Service Managed Identity", ()
         });
     });
 
+    describe("Miscellaneous", () => {
+        let managedIdentityApplication: ManagedIdentityApplication;
+        beforeEach(() => {
+            managedIdentityApplication = new ManagedIdentityApplication(
+                systemAssignedConfig
+            );
+            expect(managedIdentityApplication.getManagedIdentitySource()).toBe(
+                ManagedIdentitySourceNames.APP_SERVICE
+            );
+        });
+
+        test("ignores a cached token when claims are provided, and the Managed Identity does support token revocation, and ensures the token revocation query parameter token_sha256_to_refresh was included in the network request to the Managed Identity", async () => {
+            const sendGetRequestAsyncSpy: jest.SpyInstance = jest.spyOn(
+                networkClient,
+                <any>"sendGetRequestAsync"
+            );
+
+            let networkManagedIdentityResult: AuthenticationResult =
+                await managedIdentityApplication.acquireToken({
+                    resource: MANAGED_IDENTITY_RESOURCE,
+                });
+            expect(networkManagedIdentityResult.fromCache).toBe(false);
+            expect(networkManagedIdentityResult.accessToken).toEqual(
+                DEFAULT_SYSTEM_ASSIGNED_MANAGED_IDENTITY_AUTHENTICATION_RESULT.accessToken
+            );
+
+            const cachedManagedIdentityResult: AuthenticationResult =
+                await managedIdentityApplication.acquireToken({
+                    resource: MANAGED_IDENTITY_RESOURCE,
+                });
+            expect(cachedManagedIdentityResult.fromCache).toBe(true);
+            expect(cachedManagedIdentityResult.accessToken).toEqual(
+                DEFAULT_SYSTEM_ASSIGNED_MANAGED_IDENTITY_AUTHENTICATION_RESULT.accessToken
+            );
+
+            networkManagedIdentityResult =
+                await managedIdentityApplication.acquireToken({
+                    claims: TEST_CONFIG.CLAIMS,
+                    resource: MANAGED_IDENTITY_RESOURCE,
+                });
+            expect(networkManagedIdentityResult.fromCache).toBe(false);
+            expect(networkManagedIdentityResult.accessToken).toEqual(
+                DEFAULT_SYSTEM_ASSIGNED_MANAGED_IDENTITY_AUTHENTICATION_RESULT.accessToken
+            );
+
+            expect(sendGetRequestAsyncSpy.mock.calls.length).toEqual(2);
+            const url: URLSearchParams = new URLSearchParams(
+                sendGetRequestAsyncSpy.mock.lastCall[0]
+            );
+            expect(
+                url.has(ManagedIdentityQueryParameters.SHA256_TOKEN_TO_REFRESH)
```

Comment:
Pls assert the value of this parameter. Work with @gladjohn to ensure the spec has a test with an exact value.

E.g. : 

step 1 - get a token "my_access_token" from MSI
step 2 - get a token again (same scope and identity) and with claims

assert that the MIRP url has param name SHA256_TOKEN_TO_REFRESH set to value xyz

Reply:
msal-node uses its own constants for default MSI token, so I will assert the value of that

Reply:
done